packages/network/src/logic/ProxyStreamConnectionManager.ts

```diff
@@ -35,6 +35,7 @@ interface ProxyConnection {
     state?: State
     reconnectionTimer?: NodeJS.Timeout
     direction: ProxyDirection
+    identity?: string
```

Comment:
Make identity (i.e. userId) mandatory
```suggestion
    identity: string
```